doc_source/reference.md

```diff
@@ -4,9 +4,36 @@ The [API Reference](https://docs.aws.amazon.com/cdk/api/latest) contains informa
 
 Each library contains information about how to use the library\. For example, the [S3](https://docs.aws.amazon.com/cdk/api/latest/docs/aws-s3-readme.html) library demonstrates how to set default encryption on an Amazon S3 bucket\.
 
-## Versioning and Stability Model<a name="versioning"></a>
+## Versioning<a name="versioning"></a>
 
-Version numbers consist of three numeric version parts: *major*\.*minor*\.*patch*, and adhere to the [semantic versioning](https://semver.org) model\. This means that breaking changes to stable APIs, which is described in the next section, are limited to major releases\. Minor and patch releases are backward compatible, meaning that the code written in a previous version with the same major version can be upgraded to a newer version and be expected to continue to build and run, producing the same output\.
+Version numbers consist of three numeric version parts: *major*\.*minor*\.*patch*, and adhere to the [semantic versioning](https://semver.org) model\. This means that breaking changes to stable APIs, are limited to major releases\. Minor and patch releases are backward compatible, meaning that the code written in a previous version with the same major version can be upgraded to a newer version and be expected to continue to build and run, producing the same output\.
+
```

Comment:
I think we need an emphasis on the fact that non-stable modules (marked "experimental" or "developer preview", which is a new beast @ccfife is working on) can have breaking changes across releases, and those will be announced in our release notes under the "BREAKING CHANGES" section.

Reply:
@eladb This is mentioned below:

> Experimental
The API is still under active development and subject to non-backward-compatible changes or removal in any future version. We recommend that you do not use this API in production environments. Experimental APIs are not subject to the semantic versioning model.

What if i just link to it in this section? 

This will need a small revisit once we introduce a slightly different stability index, where CFN only API's are actually stable, and so on..

Reply:
mm scratch that, i'll try rephrasing this a bit.

Reply:
Added a note that links to the below explanation, and added the BREAKING-CHANGES info.